lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -745,6 +745,23 @@ snmp_user:
     config_get_token: ["TABLE_snmp_users", "ROW_snmp_users"]
     config_set: "%s snmp-server user %s %s %s %s %s %s"
 
+syslog_server:
+  server:
+    config_get: "show running-config all | include 'logging server'"
```

Comment:
Just to be on the safe side, can you anchor 'logging server'?

.... include '^logging server'